File: Runners/JavaRunner/java-runner/src/main/java/com/mems/manifest/ManifestDto.java
// ManifestDto.java
package com.mems.manifest;

import java.util.List;

public class ManifestDto {
    public String entrypoint;
    public Signature signature = new Signature();
    public List<HelpersBlock> helpers;
    public List<AdvancedTest> advancedTests;
    public List<SampleTest> sampleTests;
}

File: Runners/KotlinRunner/src/main/kotlin/manifest/ManifestDto.kt
package manifest

import com.fasterxml.jackson.annotation.JsonProperty

data class ManifestDto(
    @field:JsonProperty("Entrypoint")
    var entrypoint: String? = null,
    @field:JsonProperty("Signature")
    var signature: Signature = Signature(),
    @field:JsonProperty("Helpers")
    var helpers: List<HelpersBlock>? = null,
    @field:JsonProperty("AdvancedTests")
    var advancedTests: List<AdvancedTest>? = null,
    @field:JsonProperty("SampleTests")
    var sampleTests: List<SampleTest>? = null
)
